### PR TITLE
Feature/openapi for epoch v0.13.0

### PR DESCRIPTION
**Pull request by noandrea**

### CODE REVIEW

File: aeternity/signing.py [hunk 1]
```diff
@@ -106,18 +125,12 @@ def from_public_private_key_strings(cls, public, private):
         :param private: the hex encoded private key
         :return: a keypair object or raise error if the public key doesnt match
         """
-        pk = bytes.fromhex(private)
-        signing_key = SigningKey.from_string(pk, curve=SECP256k1, hashfunc=sha256)
-        kp = KeyPair(signing_key, signing_key.get_verifying_key())
+        k = bytes.fromhex(private)
+        signing_key = SigningKey(seed=k[0:32], encoder=encoding.RawEncoder)
```
Comment (e-user) [inline aeternity/signing.py]: Please add reference to https://blog.mozilla.org/warner/2011/11/29/ed25519-keys/ as to why we cut off half of the bytes.

File: aeternity/tests/test_aens.py [hunk 2]
```diff
@@ -80,22 +70,27 @@ def test_name_status_unavailable():
     assert not same_name.is_available()
 
 
+@pytest.mark.skip('skip for 0.13.0')
```
Comment (e-user) [inline aeternity/tests/test_aens.py]: Why? Add comment

File: aeternity/tests/test_api.py [hunk 3]
```diff
@@ -1,97 +1,118 @@
-from pytest import raises
+from aeternity.tests import PUBLIC_KEY
+from aeternity.epoch import EpochClient
 
-from aeternity import EpochClient
-from aeternity.epoch import AccountBalance, Transaction, CoinbaseTx, Version, EpochInfo, LastBlockInfo, BlockWithTx, \
-    AENSTransferTx, transaction_type_mapping
-from aeternity.exceptions import TransactionNotFoundException
+# from aeternity.exceptions import TransactionNotFoundException
```
Comment (e-user) [inline aeternity/tests/test_api.py]: ?

File: Jenkinsfile [hunk 4]
```diff
@@ -2,21 +2,42 @@ pipeline {
...
+    //   }
+    // }
+
+    stage('Test') {
       steps {
-        sh 'echo "Hi!"'
+        withCredentials([usernamePassword(credentialsId: 'genesis-wallet',
+                                          usernameVariable: 'WALLET_PUB',
+                                          passwordVariable: 'WALLET_PRIV')]) {
+          sh "${env.DOCKER_COMPOSE} run sdk pytest --junitxml test-results.xml aeternity/tests"
```
Comment (e-user) [inline Jenkinsfile]: Please run pylint or somesuch as part of test suite (next iteration)

File: aeternity/tests/test_contract.py [hunk 5]
```diff
@@ -23,81 +20,101 @@
...
 
 #
 # EVM
 #
 
+
 def test_evm_contract_compile():
     contract = Contract(aer_identity_contract, Contract.EVM)
     result = contract.compile()
+    print(result)
```
Comment (e-user) [inline aeternity/tests/test_contract.py]: Remove print

File: aeternity/tests/test_openapi.py [hunk 6]
```diff
@@ -0,0 +1,141 @@
+import pytest
+from aeternity.openapi import OpenAPICli
+from aeternity.tests import NODE_URL, NODE_URL_INTERNAL, PUBLIC_KEY
+
+client, priv_key, pub_key = None, None, None
+
+
+def test_generatedcli():
+    # open client
+    client = OpenAPICli("./assets/swagger/0.13.0.json", NODE_URL, NODE_URL_INTERNAL)
```
Comment (e-user) [inline aeternity/tests/test_openapi.py]: Later iteration: Make version number configurable from environment

**Comment by noandrea** [comment]
@e-user changes implemented

**Pull request merged**